salt/runners/vault.py

```diff
@@ -73,7 +78,7 @@ def generate_token(minion_id, signature, impersonated_by_master=False):
         payload = {
                     'policies': _get_policies(minion_id, config),
                     'num_uses': 1,
-                    'metadata': audit_data
+                    'meta': audit_data
```

Comment:
Can you explain why this key is changing? I didn't see a mention of this in your PR explanation. (Though I certainly could have missed it.)

Reply:
I am not sure why it was ever metadata, because the docs for vault v1 say `meta` https://www.vaultproject.io/api/auth/token/index.html#meta

Reply:
Oh, this was mentioned/changed in this PR #48585.